### DISCUSSION TITLE
Rootful Podman unable to reach host

### DISCUSSION THREAD

**Discussion by umermala** [containers/podman#23534] [Q&A]
I am running rootful podman (4.9.4-rhel). My container is on bridge network. I have a service running on host IP with port 5000. My container is unable to reach that IP on 5000 port. Anyone else faced this issue?

**Comment by afbjorklund** [comment]
You should be able to reach `host.containers.internal` (a.k.a `host.docker.internal`), but not the external host IP directly.
